new/LIT6947HymnJesus.xml

```diff
@@ -0,0 +1,74 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6947HymnJesus" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Hymn to Jesus Christ</title>
+                <title xml:lang="gez" xml:id="t2">ይትባረክ፡ እግዚአብሔር፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Yǝtbārak ʾƎgziʾabǝḥer</title>
+                <title xml:lang="en" corresp="#t2">The Lord shall be praised</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="EMIP02368"/>
+                    <witness corresp="BLorient8814"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Hymn of 13 stanzas, each beginning with the phrase <foreign xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡</foreign>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-03-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div xml:lang="gez" type="edition">
+                <note>The following incipit is taken from the first stanza from <ref type="mss" corresp="EMIP02368"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>
+                        ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡ 
```

Comment:
It is possible to encode verses with `<l="1">ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡</l>`

Reply:
XML does not allow `<ab>` to be followed by `<l>`. How can I resolve?
```
<div type="textpart" subtype="incipit">
                    <ab>
                    <l="1">ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡</l>    
                    <l="2">እስራኤል፡ ወይሁዳ፡ እንተ፡ ኃለፉ፡ በውሥጣ፡</l>
                    <l="3">በክቡር፡ ደሙ፡ ለነፍስየ፡ ዘተሣየጣ፡</l>
                    <l="4">ካዕበኒ፡ በሰበነ፡ ርእሱ፡ ዘሦጣ፡</l>
                    <l="5">ለማርያም፡ ንጽሕት፡ አፍአ፡ ወውሥጣ። </l> 
                    </ab>
                </div>
```

Reply:
by not using `ab `when you have` l`

Reply:
But the first line with `<l>` is still marked with a red line.

```
<div xml:lang="gez" type="edition">
                <note>The following incipit is taken from the first stanza from <ref type="mss" corresp="EMIP02368"/>.</note>
                <div type="textpart" subtype="incipit">
                    <l="1">ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡</l>    
                    <l="2">እስራኤል፡ ወይሁዳ፡ እንተ፡ ኃለፉ፡ በውሥጣ፡</l>
                    <l="3">በክቡር፡ ደሙ፡ ለነፍስየ፡ ዘተሣየጣ፡</l>
                    <l="4">ካዕበኒ፡ በሰበነ፡ ርእሱ፡ ዘሦጣ፡</l>
                    <l="5">ለማርያም፡ ንጽሕት፡ አፍአ፡ ወውሥጣ። </l> 
                </div>
            </div>
```

Reply:
Sorry, my typo. I meant  `<l n="1">`